### PR TITLE
[Banner] - replace banner on main page

### PR DESCRIPTION
**Pull request by pietro-maximoff**
https://sovryn.monday.com/boards/2218344956/pulses/2714824791

### CODE REVIEW

File: src/app/pages/LandingPage/components/Banner/GetStartedBanner.tsx [hunk 1]
```diff
@@ -1,24 +1,29 @@
...
-        alt={t(translations.landingPage.banner.getStarted)}
-      />
+    <div className="tw-flex tw-flex-col tw-justify-center sm:tw-flex-row tw-items-end tw-relative tw-w-full tw-h-full">
+      <Picture src={getStartedBanner} srcSet={pictureSrcSet} />
       <Button
-        className="tw-relative sm:tw-absolute tw-self-center sm:tw-self-auto tw-mb-6 tw-ml-0 tw-mt-2 sm:tw-mt-0 sm:tw-ml-8"
+        className="tw-absolute tw-self-center tw-min-h-8 sm:tw-min-h-12 sm:tw-self-auto tw-bottom-3 sm:tw-bottom-6 tw-text-sm sm:tw-text-xl tw-border-2 tw-py-1 sm:tw-py-3"
         text={t(translations.landingPage.banner.getStarted)}
-        href="/buy-sov"
+        href="/zero"
```
Comment (tiltom) [inline src/app/pages/LandingPage/components/Banner/GetStartedBanner.tsx:24]: We have a constant `zeroUrl` in `classifiers.ts`, pls use it instead so we are consistent.

**Review by tiltom — CHANGES_REQUESTED** [review]

**Comment by tiltom** [comment]
I have one more question - when do we want to deploy it? I suppose we want to do it pretty soon, so I would target `master` instead, what do you think @soulBit ?

File: src/app/pages/LandingPage/components/Banner/GetStartedBanner.tsx (hunk 1 above)
Comment (pietro-maximoff) [reply src/app/pages/LandingPage/components/Banner/GetStartedBanner.tsx:24]: thanks, good to know

**Comment by Alitasovryn** [comment]
Screen Shot 2022-06-02 at 8 19 48 AM

The Zero waitlist link is not getting redirected to [https](https://test.sovryn.app/zero) as mentioned in the story spec

**Comment by pietro-maximoff** [comment]
> Screen Shot 2022-06-02 at 8 19 48 AM
> 
> The Zero waitlist link is not getting redirected to [https](https://test.sovryn.app/zero) as mentioned in the story spec

@Alitasovryn pls check again

File: src/app/containers/PageContainer/components/DefaultHeaderComponent/DefaultHeaderComponent.tsx [hunk 2]
```diff
@@ -40,6 +40,13 @@ import { bignumber } from 'mathjs';
 const showPerps = !isMainnet || isStaging;
 const showZero = isMainnet || isStaging;
 
+interface PagesProps {
```
Comment (tiltom) [inline src/app/containers/PageContainer/components/DefaultHeaderComponent/DefaultHeaderComponent.tsx:43]:
```suggestion
type PageProps = {
```

Comment (tiltom) [reply src/app/containers/PageContainer/components/DefaultHeaderComponent/DefaultHeaderComponent.tsx:43]: Just to clarify - you can use either `interface` or `type` but if you wanted it to be an `interface`, it should've had a prefix (`I`).

**Review by tiltom — CHANGES_REQUESTED** [review]

**Pull request closed without merging**